src/Engine/GameObject/GameObject.cpp

```diff
@@ -23,7 +23,7 @@ GameObject::GameObject(double x_pos, double y_pos, double x_vel, double y_vel, i
 }
 
 GameObject::~GameObject() {
-    LOG(INFO) << "GameObject with id: " << id_ << " died";
+    //LOG(INFO) << "GameObject with id: " << id_ << " died";
```

Comment:
Instead of commenting out, maybe put it as debug.

Reply:
http://rpg.ifi.uzh.ch/docs/glog.html
There is no debug severity level :(

Reply:
Oh just kidding. 

DLOG(INFO) i think...